RadixWallet/Features/Signing/Signer.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Sargon
+
+// MARK: - Signer
```

Comment:
Just moved out from `AccountOrPersona` file. Ought to be moved into Sargon later.